<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmWaveDouble</class>
 <widget class="QWidget" name="frmWaveDouble">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="WaveDouble" name="waveDouble1"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble2"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble3"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble4"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble5"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble6"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble7"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble8"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble9"/>
   </item>
   <item>
    <widget class="WaveDouble" name="waveDouble10"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WaveDouble</class>
   <extends>QWidget</extends>
   <header>wavedouble.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
